types/express-unless/index.d.ts

```diff
@@ -11,11 +12,14 @@ declare function unless(options: unless.Options): express.RequestHandler;
 declare function unless(options: unless.Options["custom"]): express.RequestHandler;
 
 declare namespace unless {
+    type pathFilter = string | RegExp | { url: string | RegExp, methods?: string[] };
```

Comment:
the object form can also have `method?: string | string[]` like the top-level option
https://github.com/jfromaniello/express-unless/blob/master/index.js#L25

Reply:
`url` can also be a `RegExp` https://github.com/jfromaniello/express-unless/blob/master/index.js#L69

Reply:
You are right about `method` optional property in object form, I will fix that. But what do you mean in your second comment? In my declarations there is:
`type pathFilter = string | RegExp | { url: string | RegExp, methods?: string[] };`
So in object form `url` can either be a `string` or a `RegExp`.